algosdk/future/transaction.py

```diff
@@ -1175,13 +1176,14 @@ class ApplicationCallTxn(Transaction):
         accounts (list[str])
         foreign_apps (list[int])
         foreign_assets (list[int])
+        extra_pages (int)
     """
 
     def __init__(self, sender, sp, index,
                  on_complete, local_schema=None, global_schema=None,
                  approval_program=None, clear_program=None, app_args=None,
                  accounts=None, foreign_apps=None, foreign_assets=None,
-                 note=None, lease=None, rekey_to=None):
+                 note=None, lease=None, rekey_to=None, extra_pages=0):
```

Comment:
nit: can you move extra_pages to be before `note`, `lease`, and `rekey_to`. It is convention for those three to appear last in the parameter list.

Reply:
I don't think we can change the order of arguments.  It would silently break existing code.  Unless we introduced a * early in the list, which would require the use of keyword args.  Then the code would break, but not silently.

Reply:
Ah yes, I forgot about that. Well if this SDK requires putting the extra pages field last, we should probably put it last across all the SDKs to be consistent.